sunpy_sphinx_theme/sunpy/navbar.html

```diff
@@ -38,10 +38,10 @@
                   <a href="http://docs.sunpy.org/projects/irispy/en/latest/">IRISPy</a>
```

Comment:
I am wondering if we should pull IRISPy as well. At the very least it should be at the end.

Reply:
We can move it to the bottom